dali/python/backend_impl.cc

```diff
@@ -541,6 +542,7 @@ TFValue ConvertTFRecordDefaultValue(TFFeatureType type, py::object val) {
 #endif  // DALI_BUILD_PROTO3
 
 PYBIND11_MODULE(backend_impl, m) {
+  dali::InitOperatorsLib();
```

Comment:
Why here and not in the pipeline?

Reply:
I am afraid we need to do this at a lower level (as Janusz suggests) if we want the C++ API to behave correctly

Reply:
We _don't want_  dali pipeline do depend on operators - I thought that was the idea all along, that we can have a DALI pipeline, but not use all the operators - something we've seen DALI used for already.